code/API_definitions/qod-api.yaml

```diff
@@ -328,6 +477,166 @@ components:
+          description: The target minimum data rate.  Set to zero when no target minimum is defined in this profile.
+        maxUpstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The maximum best effort data
+            Equal to targetMinimumUpstreamRate when max is the as the target. 
+        maxUpstreamBurstRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The burst rate for best effort data
+        targetMinDownstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: The target minimum data rate.  Set to zero when no target minimum is defined in this profile.
+        maxDownstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The maximum best effort rate
+            Equal to targetMinimumUpstreamRate when max is the as the target. 
+        maxDownstreamBurstRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The maximum best effort burst rate
+            Equal to targetMinimumUpstreamRate when max is the as the target. 
+        minDuration:
+          $ref: "#/components/schemas/Duration"
+          description: The shortest time period that is supported this profile can be deployed.
+        maxDuration:
+          $ref: "#/components/schemas/Duration"
+          description: The maximum length of time that this profile can be deployed.
+        priority:
+          type: number
+          example: 5.0
+          description: |
+            Priority for this session
+          format: float
+          minimum: 0
+          maximum: 10
+          default: 5
+        packetDelayBudget:
+          $ref: "#/components/schemas/Duration"
+          description: The maximum latency for a packet.
+        jitter:
+          $ref: "#/components/schemas/Duration"
+          description: |
+            Jitter refers to the variation in the time it takes for none queue building
+            packets to travel across a network measured in milliseconds (ms). In terms of
+            maximum deviation for round trip latency on a network, jitter can be defined
+            as the difference between the highest and lowest latency values experienced
+            by the 99th percentile of traffic.
+          
+            To be more specific, if we consider the round trip latency values of the 99th
+            percentile of traffic, jitter is the measure of how much these values deviate
+            from the average round trip latency for this percentile. A low jitter value
+            indicates that the latency values are relatively consistent, whereas a high
+            jitter value implies that there is a significant variation in latency times,
+            which can negatively impact the performance of real-time applications such as
+            VoIP, video conferencing, and online gaming.
+        packetErrorLossRate:
+          type: integer
```

Comment:
`description: The Packet Error Loss Rate is the fraction of total packets sent that are either not delivered to their destination at all, or experience a delay greater than the allowed Packet Delay Budget`

Not sure why PELR should be expressed as 2^(-N) rather than 10^(-N). 3GPP specified PELRs are all 10^(-N), so having to convert these to powers of 2 is inconvenient. But maybe other standards use powers of 2.